tests/baselines/reference/undefinedTypeAssignment3.errors.txt

```diff
@@ -1,8 +0,0 @@
-undefinedTypeAssignment3.ts(1,5): error TS2397: Declaration name conflicts with built-in global identifier 'undefined'.
-
-
-==== undefinedTypeAssignment3.ts (1 errors) ====
-    var undefined = null;

```

Comment:
I'm brainfarting and failing to find the right function to still emit this error on non-types. Will figure out later when my brain is working...